modules/apache-airflow/templates/values.yaml

```diff
@@ -806,7 +810,9 @@ scheduler:
   extraVolumeMounts: []
 
   # Select certain nodes for airflow scheduler pods.
-  nodeSelector: {}
+  nodeSelector: {
+    spotinst.io/node-lifecycle: "od"
```

Comment:
Maybe for later we can think about other spot setup configuraitons that would make Airflow more resilient so we actually gain the benefit of the spot instances.

Reply:
For airflow I think we can take advantage of this if we move over to kubernetes workers instead of celery works. The reason is that the individual tasks can run on spot instances, but everything to kick off those tasks don't do well to be interrupted on spot instances.